src/pages/en/reference/cli-reference.md

```diff
@@ -37,6 +37,28 @@ Runs diagnostics (such as type-checking) against your project and reports errors
 
 This command is intended to be used in CI workflows.
 
+### astro telemetry
+
+Sets telemetry configuration for the current user. Telemetry is anonymous data that provides insights into what features are most often used.
+
+Telemetry can be disabled with this cli command:
+
+```shell
+astro telemetry disable
+```
+
+Telemetry can later be re-enabled with:
+
+```shell
+astro telemetry enable
+```
+
+The `clear` command resets the telemetry data:
+
+```shell
+astro telemetry clear
+```
+
```

Comment:
As per https://github.com/withastro/astro/pull/3256#issuecomment-1115219301, maybe worth flagging here how to handle CI:

```suggestion

> Want to disable telemetry in CI environments? Make sure you add the `astro telemetry disable` command to your CI scripts.

```